kDriveCore/Data/Cache/DriveFileManager.swift

```diff
@@ -51,13 +51,58 @@ public final class DriveFileManager {
             DropBoxSize.self,
             DropBoxValidity.self
         ]
+
         private let fileManager = FileManager.default
+
+        // MARK: appDirectory URL
+
+        /// App files url
+        public var appDirectoryURL: URL? {
```

Comment:
I added more directory URLs in order to be consistent with the current abstraction we use